README.md

```diff
@@ -18,3 +18,17 @@ GST_PLUGIN_PATH=Release:$GST_PLUGIN_PATH gst-launch-1.0 \
     video/x-raw, width=1920, height=1080, framerate=60/1 ! cefdemux name=d d.video ! queue ! videoconvert ! \
     xvimagesink d.audio ! queue ! audioconvert ! pulsesink
 ```
+
+Record website video + audio (with audiomixer)
+
+``` shell
+gst-launch-1.0 cefsrc url="https://soundcloud.com/platform/sama" ! \
+    video/x-raw, width=1920, height=1080, framerate=60/1 ! \
+    cefdemux name=demux ! videoconvert ! queue max-size-bytes=0 max-size-buffers=0 max-size-time=3000000000 ! x264enc ! \
```

Comment:
perhaps reindent so this still looks tidy

Reply:
I have tried to split by concerns at least from how I see it

Reply:
But ye it does end up too long. Split it in more lines